Formula/k/kubetui.rb

```diff
@@ -0,0 +1,41 @@
+class Kubetui < Formula
+  desc "TUI tool for monitoring and exploration of Kubernetes resources"
+  homepage "https://github.com/sarub0b0/kubetui"
+  url "https://github.com/sarub0b0/kubetui/archive/refs/tags/v1.5.0.tar.gz"
+  sha256 "137ef45cb539a56b16990c72ae60c7247d4ff90be3c16eec029dbd5e0480e677"
+  license "MIT"
+  head "https://github.com/sarub0b0/kubetui.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/kubetui --version")
+
+    cmd = "#{bin}/kubetui --kubeconfig not_exist 2>&1"
+
+    if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
+      # Use script command because it fails with:
+      #   failed to enable raw mode: Os { code: 6, kind: Uncategorized, message: "No such device or address" }
+      ohai  cmd
+
+      r, w, pid = PTY.spawn("script -q -e -c \"#{cmd}\" screen.log")
```

Comment:
Why not just run the command directly?

Reply:
Thank you for the review.
kubetui needs to change the terminal to raw mode, but Linux CI does not have a tty. Therefore, I have worked around this issue by using the script command.

https://github.com/actions/runner/issues/241

Reply:
But you're also running it in a pseudo tty. Doesn't that work?

Reply:
Is it with `PTY.spawn`?

Reply:
Yes, pty is pseudo tty

Reply:
I tried using PTY, and it worked successfully. I have pushed the modified code, so please review it.